tooling/miniutil/src/fmt/ty.rs

```diff
@@ -89,8 +89,8 @@ fn fmt_pointee_info(pointee: PointeeInfo) -> String {
         false => ", uninhabited",
     };
     let freeze_str = match pointee.freeze {
-        true => ", freeze",
-        false => "",
+        true => "",
+        false => ", unfreeze",
```

Comment:
If you think always printing `freeze` is too verbose, please use `!Freeze` (and `!Unpin`) to make things slightly more standard-looking.